examples/cmake/test_uct.c

```diff
@@ -0,0 +1,16 @@
+#include <uct/api/uct.h>
+#include <assert.h>
+
+int main(int argc, char **argv)
+{
+    ucs_async_context_t *async;
+    uct_worker_h worker;
+
+    /* Initialize context */
+    ucs_status_t status = ucs_async_context_create(UCS_ASYNC_MODE_THREAD_SPINLOCK, &async);
+    assert(UCS_OK == status);
+
+    /* Create a worker object */
+    status = uct_worker_create(async, UCS_THREAD_MODE_SINGLE, &worker);
```

Comment:
pls destroy the created objects and return 0

Reply:
Added